regionServiceClient/lib/cloudregister/registerutils.py

```diff
@@ -45,8 +45,11 @@ def add_hosts_entry(smt_server):
     smt_hosts_entry_comment += 'retain comment as well\n'
     hosts = open('/etc/hosts', 'a')
     hosts.write(smt_hosts_entry_comment)
+    smt_ip = smt_server.get_ipv4()
```

Comment:
Might as well put this in an else clause. So it's not called when has_ipv6_access is True.